pages/workspace/tracking-plan/properties.mdx

```diff
@@ -22,9 +22,68 @@ relevant for that specific event, when you analyze your data. You might be want
 
 Event properties are sent with the event in the <Link passHref href="/workspace/tracking-plan/events#a-namelog-eventa-log-event"><a>event call</a></Link>.
 
-### <a name="user-properties"></a> User Properties
+## <a name="group-properties"></a> Group properties
 
-User properties to describe the state of your user. It is information that is
+Group properties describe the state of a group you defined previously.
+
+Similar to User properties, this information is usually relevant in a larger scope than for exactly that event.
+
+> 💡 **When should I use Groups and Group Properties?**
+>
+> <Link passHref href="/data-design/groups">
+>   <a>
+>     Read more here about how Group Properties work and why they may be the
+>     right choice for you
+>   </a>
+> </Link>
+> .
+
+**Example**
+
+For B2B companies, products are used by multiple companies, each consisting of a number of users.
+In that case, you can maintain the state of each company with group properties, increase the count of employees when a new user joins a particular company for example.
+
+**Functionality**
+
+Group properties are updated with the event in the <Link passHref href="/workspace/tracking-plan/events#a-nameupdate-groups"><a>Update Groups</a></Link> action.
```

Comment:
aaaaa. I'm allergic to these broken docs links (i'm always sharing docs links with prospects and customers and _if_ you manage to parse the actual article name from the docs link, then following that, this link says to me "eew. what's wrong with their docs links?")

I believe in most cases there's a way to bypass this. I think just removing the "a-name" part will probably just work.

TODO fix this link.

Reply:
Mmmm, it's because we haven't shipped them yet 😅